core/domain/feedback_domain_test.py

```diff
@@ -161,3 +162,68 @@ def test_to_dict(self) -> None:
         self.assertDictEqual(
             observed_feedback_message_reference.to_dict(),
             expected_feedback_message_reference)
+
+
+class FullyQualifiedMessageIdentifierDomainUnitTests(
+    test_utils.GenericTestBase):
+
+    def setUp(self) -> None:
+        super(FullyQualifiedMessageIdentifierDomainUnitTests, self).setUp()
+        self.exp_id = 'exp'
+        self.message_id = 1
+        self.thread_id = 'exp.thread'
+
+    def test_initialize(self) -> None:
+        message_identifier = feedback_domain.FullyQualifiedMessageIdentifier(
+                self.thread_id, self.message_id
+            )
+        self.assertEqual(
+            message_identifier.message_id,
+            self.message_id
+        )
+        self.assertEqual(
+            message_identifier.thread_id,
+            self.thread_id
+        )
+
+
+class FeedbackThreadSummaryDomainUnitTests(test_utils.GenericTestBase):
+
+    def setUp(self) -> None:
+        super(FeedbackThreadSummaryDomainUnitTests, self).setUp()
+        self.exp_id = 'exp'
+        self.message_id = 'message'
+        self.thread_id = 'exp.thread'
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL) # type: ignore[no-untyped-call]
+        self.exploration = self.save_new_valid_exploration(self.exp_id, self.owner_id) # type: ignore[no-untyped-call]
+        self.thread_id = feedback_services.create_thread( # type: ignore[no-untyped-call]
```

Comment:
@nithusha21 flagging this for you to review. Is ignoring types here okay?

Reply:
I don't understand why we need to ignore types. Shouldn't be necessary IMO. Can you explain why the types need to be ignored?

Reply:
@Hatilar420 can we avoid ignoring types here?

Reply:
![mypy_checks_failing](https://user-images.githubusercontent.com/29446639/155290799-268414d9-f156-4df5-83e5-53b8c6e6ded7.png)

@U8NWXD  @nithusha21  the methods i am using here are untyped (return type is not specified) , which are resulting in the failure of mypy checks. to avoid that i am ignoring the types. 
 
is there any other way , please let me know ?

Reply:
@Hatilar420 can you add types to the functions you're calling (i.e. go to where `get_user_id_from_email` is defined and add a return type annotation)?